corehq/apps/users/migrations/0046_add_default_mobile_worker_role.py

```diff
@@ -0,0 +1,36 @@
+# Adds the default mobile worker role to existing projects that don't yet have one.
+from django.db import migrations
+
+from corehq.apps.domain.models import Domain
+from corehq.apps.users.models import Permissions
+from corehq.apps.users.models_role import UserRole
+from corehq.apps.users.role_utils import UserRolePresets
+from corehq.util.django_migrations import skip_on_fresh_install
+
+
+@skip_on_fresh_install
+def _add_default_mobile_worker_role(apps, schema_editor):
+    for domain in Domain.get_all():
+        items = UserRole.objects.filter(domain=domain, is_commcare_user_default=True)
```

Comment:
You might be better off doing `UserRole.objects.filter(is_commcare_user_default=True)` outside of the loop, rather than doing a query for each domain.